### PR TITLE
docs(getting-started.md): use -D instead of -g

### PR DESCRIPTION
**Pull request by kongou-ae**
This PR changes global install to local install in the installation of textlint. This PR is related to #191

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ You can find a rule in [A Collection of textlint rule](https://github.com/textli
 As an example, let's install [textlint-rule-no-todo](https://github.com/azu/textlint-rule-no-todo "textlint-rule-no-todo").
 
 ```sh
-$ npm install -g textlint-rule-no-todo
+$ npm install -D textlint-rule-no-todo
```
Comment (azu) [inline docs/getting-started.md]: `--save-dev` looks like better.

File: docs/getting-started.md [hunk 2]
```diff
@@ -13,7 +13,7 @@
 You can install `textlint` using npm:
 
 ```sh
-$ npm install -g textlint
+$ npm install -D textlint
```
Comment (azu) [inline docs/getting-started.md]: Can you add `npm init` before this command or add "Creat workspace" secion?
Getting Started aim to support begginer user who is also noder.js begginer.

```sh
# Create package.json
$ npm init --yes

# Installed textlint locally
$ npm install --save-dev textlint
```

OR

	# Create new workspace	

	Create `<your-workspace>` directory and create `package.json` file.
	A [package.json](https://docs.npmjs.com/files/package.json "package.json") manage dependencies of packages that include `textlint`.		
	
	```sh
	# Create your workspace direactory and move it
	mkdir your-workspace
	cd your-workspace

	# `npm init` command create `package.json` file
	npm init --yes
	```
	
	## Installation
	
	...
	
	```sh
	# Installed textlint locally
	$ npm install --save-dev textlint
	```

File: docs/getting-started.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ $ npm install -g textlint-rule-no-todo
 You can run textlint on any Markdown files:
 
 ```sh
-$ textlint --rule no-todo file.md
+$ $(npm bin)/textlint --rule no-todo file.md
```
Comment (azu) [inline docs/getting-started.md]: We want to write cross platform way for `$(npm bin)/`.

**Comment by kongou-ae** [comment]
Thank you for review! I will try to fix these.

File: package.json [hunk 4]
```diff
@@ -34,6 +34,7 @@
     "prettier": "prettier --write 'packages/**/*.{js,jsx,ts,tsx,css}'"
   },
   "devDependencies": {
+    "cross-env": "^5.1.3",
```
Comment (azu) [inline package.json]: Is `cross-env` needed?

File: docs/getting-started.md [hunk 5]
```diff
@@ -2,31 +2,50 @@
...
+cd your-workspace
+
+# `npm init` command creates `package.json` file.
+npm init --yes
+```
+
+### Installation of textlint
    
-You can install `textlint` using npm:
+You can install `textlint` using npm. We recommend installing `textlint` locally:
```
Comment (0x6b) [inline docs/getting-started.md]: Lengthy a bit, but how about this?

> We recommend installing `textlint` locally, which means installing it in the `<your-workspace>/node_modules` folder, by running `npm` command with `--save-dev` option.

I'm afraid non-developer textlint user could not connect "installing locally", "`--save-dev`" option, and "`node_modules` directory" created after the command.

File: docs/getting-started.md [hunk 6]
```diff
@@ -2,31 +2,50 @@
...
 ```
 
-## Installation of rules
+### Installation of rules
 
-You can find a rule in [A Collection of textlint rule](https://github.com/textlint/textlint/wiki/Collection-of-textlint-rule "A Collection of textlint rule")
+You can find a rule in [A Collection of textlint rule](https://github.com/textlint/textlint/wiki/Collection-of-textlint-rule "A Collection of textlint rule").
 
-As an example, let's install [textlint-rule-no-todo](https://github.com/azu/textlint-rule-no-todo "textlint-rule-no-todo").
+As an example, let's install [textlint-rule-no-todo](https://github.com/azu/textlint-rule-no-todo "textlint-rule-no-todo"). We recommend installing a rule locally:
```
Comment (azu) [inline docs/getting-started.md]: > We recommend installing a rule locally:

Following is better, I think.

> If you have installed `textlint` locally, you must install each rule locally as well.

**Comment by 0x6b** [comment]
[Travis CI failure](https://travis-ci.org/textlint/textlint/jobs/327221974#L1286) looks temporary error. Will restart the job.

> no-dead-link: https://lernajs.io/ is dead. (522 Origin Connection Time-out)

File: docs/getting-started.md (hunk 5 above)
Comment (0x6b) [reply docs/getting-started.md]: I should mention at first that thank you very much for your efforts, @kongou-ae!

File: package.json (hunk 4 above)
Comment (kongou-ae) [reply package.json]: Sorry... This is my mistake. I fixed it.

**Comment by 0x6b** [comment]
Restarting job at Travis CI solved the build failure. AppVeyor failed with timeout as well but another push will solve it.

**Comment by azu** [comment]
FYI: I've created an issue about timeout error.

> Stability of textlint-rule-no-dead-link #453 

This timeout error can be skip in this issue.

File: docs/getting-started.md (hunk 6 above)
Comment (kongou-ae) [reply docs/getting-started.md]: I fixed this.

File: docs/getting-started.md (hunk 5 above)
Comment (kongou-ae) [reply docs/getting-started.md]: I fixed this. I used more simply sentence. Please check it whether my change matches your proposal or not.

File: docs/getting-started.md [hunk 7]
```diff
@@ -26,7 +26,7 @@ npm init --yes
 
 ### Installation of textlint
    
-You can install `textlint` using npm. We recommend installing `textlint` locally:
+You can install `textlint` using npm. We recommend that you install `textlint` locally by running npm command with `--save-dev` option. This means that npm installs `textlint` in the <your-workspace>/node_modules folder.
```
Comment (0x6b) [inline docs/getting-started.md]: Please enclose &lt;your-workspace&gt;/node_modules with backtick (\`). Without backtick, GitHub thinks &lt;your-workspace&got; as unknown tag and strip it from rendered HTML. Other than that, the modification looks good to me!

Comment (kongou-ae) [reply docs/getting-started.md]: Thanks! I fixed it.

**Comment by 0x6b** [comment]
LGTM

**Comment by 0x6b** [comment]
I found @azu approved the changes. Will merge this pull request.

**Comment by 0x6b** [comment]
@kongou-ae, thank you very.much for your help. Congrats to your first contribution 😄

**Comment by 0x6b** [comment]
Closed the issue. Thank you for pointing out.

**Pull request merged**